### PR TITLE
Managed Instance failover swagger

### PR DESCRIPTION
**Pull request by milanbrkic-ms**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by NullMDR** [comment]
@MilanBrkicFON Any update on this PR?

**Comment by MilanBrkicFON** [comment]
@NullMDR Last week changes landed in production, but I didn't have time to update this pr.
Now is ready for review!

**Comment by MilanBrkicFON** [comment]
Hi @NullMDR are you ok with the changes? Can we merge this PR?

**Comment by NullMDR** [comment]
@MilanBrkicFON You need to wait for ARM review. I've emailed them.

**Comment by MilanBrkicFON** [comment]
@pilor @allenjzhang Can I have this PR reviewed please?

**Comment by pilor** [comment]
> @pilor @allenjzhang Can I have this PR reviewed please?

API reviews are a weekly rotation described [here](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review.html#api-review-on-call).  Please reach out to this week or next week's reviewer

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json
Comment (allenjzhang) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json:53]: nit: It will be good to move these standard parameters up to line 30 and group them together before the body parameters.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json
Comment (allenjzhang) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json:105]: There are many unused and duplicate definitions here. They should be removed. I looked over other json in SQL, seems there are copy and paste since they exhibit same pattern. This creates problems in our codegen(autorest) and producing duplicate class schemas. 

I opened an [issue](https://github.com/Azure/azure-rest-api-specs/issues/9640) on our side to work with you to clean up these swaggers in the future.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json
Comment (allenjzhang) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json:165]: nit: We would like to standardize `security` and `securityDefinitions` at beginning of the file to create consistency layout and ease of review.

**Review by allenjzhang — CHANGES_REQUESTED** [review]

**Comment by allenjzhang** [comment]
@MilanBrkicFON, pretty straightforward PR, though I have noticed consistent pattern issues with SQL swagger files. Please take a look at these easy fixes. Will loop back to work with you to cleanup if these are manually created. If these are generated, please incorporate feedback and make appropriate changes to generate higher quality swagger in the future.

BTW, this is SDK team review. As per policy, ARM folks need to sign off for new APIs. Giving the low complexity of this PR, it should be a quick sign off. Please engage ARM folks per @pilor's instruction.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json
Comment (allenjzhang) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json:28]: nit: You don't have to re-define your own standard parameters. Please reference the ones under common-type\resource-management\v1\types.json. Since there are many instances in the current api-version folder, you may defer to cleanup time. But should at least standardize to single common definition in your RP as the previous comment indicated.

**Comment by MilanBrkicFON** [comment]
@allenjzhang Thanks for the review. I think I've addressed all of your comments. Can you please check again?
I will also update internal docs on ARM development to warn engineers what we are doing wrong and to avoid same mistakes in the future, but I think the right solution here would be working auto-generation script.

**Review by pilor — APPROVED** [review]
:shipit:

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json
Comment (MilanBrkicFON) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json]: Done.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json
Comment (MilanBrkicFON) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json:49]: I was getting 

> The subscription ID that identifies an Azure subscription.
, ---
The ID of the target subscription.
).
  Error: '$.parameters.SubscriptionIdParameter.description' has incompatible values (---
The subscription ID that identifies an Azure subscription.
, ---
The ID of the target subscription.
).
>
when I referenced `common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter` and didn't know how to fix that.

Comment (allenjzhang) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json:49]: Can you try update this to common-type AND remove the subscription id definition below on Line 95? I think that should do it. In the meantime, I remove the change requested status. Thx

**Review by allenjzhang — APPROVED** [review]

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json
Comment (MilanBrkicFON) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json:49]: I did try, but I was getting the error when execute `autorest specification\sql\resource-manager\readme.md --azure-validator=true` (I pasted error in comment above).
Can we just leave it as is? I would really appreciate to merge this changes, so I could continue with client SDKs

**Comment by MilanBrkicFON** [comment]
@pilor @allenjzhang @NullMDR Can someone merge this PR please? I am not authorized to do it.

**Pull request merged**
